src/io/sb3/fromSb3.ts

```diff
@@ -198,7 +213,7 @@ function getBlockScript(blocks: { [key: string]: sb3.Block }) {
                 }) as Block
```

Comment:
Is the `// TODO: Set "parent"` above this line something that should be taken care of?